### PR TITLE
Changelog: prepare for the release of version 1.2.0

### PR DESCRIPTION
**Pull request by jrfnl**
ℹ️  Note: this PR is in draft as PR #107 still needs to be merged and included in the 1.2.0 release.

---

### Changelog: import changelog for v 1.0.2

The changelog for v 1.0.2 only existed in the 1.0.2 tag.

Includes adding dates to undated release entries.

Ref:
* ddd1e0e
* e9df81e

### Changelog: import changelog for v 1.1.0

The changelog for v 1.1.0 only existed in the 1.1.0 tag.

Ref:
* 1ea57a0
* 2fc9da3
* f66106f

### Changelog: add changelog for the 1.2.0 release 

The release date has been tentatively set to next **Monday, July 19, 2021**.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,53 @@ All notable changes to this project will be documented in this file.
...
+:warning: This version drops support for PHP 5.2!
+
+### Changed
+* PHP 5.2 is no longer supported. The minimum supported PHP version for the WHIP library is now PHP 5.3. [#96]
+* The previous solution to prevent duplicate messages as included in v1.0.2 has been improved upon and made more stable. Props [Drew Jaynes]. [#44]
+* The `Whip_InvalidOperatorType::__construct()` method now has a second, optional `$validOperators` parameter. [#62]
+    If this parameter is not passed, the default set of valid operators, as was used before, will be used.
+* Improved protection against XSS in localizable texts. [#50]
+* Improved support for translating localizable texts (I18n). [#59]
+* The distributed package will no longer contain development related files. [#45]
```
Comment (IreneStr) [inline CHANGELOG.md:18]:
```suggestion
* The distributed package will no longer contain development-related files. [#45]
```

Comment (jrfnl) [reply CHANGELOG.md:18]: Happy to change it, but the dashes are a "dutchism" and not expected in English.

Comment (IreneStr) [reply CHANGELOG.md:18]: Grammarly proposes the hyphen as well. Also see the section about `Compound adjectives` here: https://www.lexico.com/grammar/hyphen

Comment (jrfnl) [reply CHANGELOG.md]: Fair enough. In that case: done. 😁

**Pull request merged**